### ISSUE TITLE
Errors while trying to build `license.gradle` missing 

### ISSUE THREAD

**Issue by BarakStout** [TAK-Product-Center/Server#4]
Cloned and ran `cd src && ./gradlew clean bootWar`. Got the error message `> Could not read script '/Users/barakstout/Downloads/TAK/Server/src/gradle/license.gradle' as it does not exist.`. Tried it on Intel and M1 Macs and an Ubuntu machine. See screenshots below: 

<img width="1051" alt="Screen Shot 2022-06-28 at 3 56 17 PM" src="https://user-images.githubusercontent.com/34528865/176277633-594bbf0b-35c9-4d49-8891-ef0773d46940.png">
<img width="1064" alt="Screen Shot 2022-06-28 at 3 54 55 PM" src="https://user-images.githubusercontent.com/34528865/176277637-36fd3c14-3152-4b1a-88f7-4cb932f0ede7.png">

**Comment by pratimakulkarni** [comment]
Yes even I am getting same error. When trying to build via command line or eclipse Ide.
Using  "gradlew.bat clean bootWar" as I am using windows.
Error in below attached image.

![cmd_nCY3dKD3bw](https://user-images.githubusercontent.com/26931278/176658522-61a850e8-056d-4e65-a6ff-e181b8d9c73d.png)

**Comment by RusselTakservya** [comment]
I resolved this issue on Ubuntu 20 by using 'git clone' to grab the project rather than downloading and unpacking the zip file through the github webpage.

**Comment by BarakStout** [comment]
Wired. I tried it now and I am getting new errors now on mac. Will try on ubuntu. 

<img width="942" alt="Screen Shot 2022-07-05 at 3 11 13 PM" src="https://user-images.githubusercontent.com/34528865/177398894-46fb2a59-1f9b-4e5c-9f5c-2e9f6d0a23c5.png">

**Comment by BarakStout** [comment]
I get the same error on a fresh install of Ubuntu 21

**Comment by mighty-swede** [comment]
Check if you got all tags. If no tag, tag it och fetch tag
